lib/hammer_cli_katello/hostgroup_extensions.rb

```diff
@@ -20,6 +20,16 @@ def self.included(base)
         _('Organization name to search by'), attribute_name: :option_organization_name
       base.option '--query-organization-label', 'ORGANIZATION_LABEL',
         _('Organization label to search by'), attribute_name: :option_organization_label
+
+      base.validate_options do
+        organization_options = [
```

Comment:
can you reuse https://github.com/Katello/hammer-cli-katello/blob/master/lib/hammer_cli_katello/organization_options.rb#L12 here ?